hastexo/tasks.py

```diff
@@ -99,7 +107,8 @@ def launch_stack(self, heat, stack_name, stack_template):
             status = stack.stack_status
             retry += 1
             if retry >= self.retries:
-                status = 'CREATE_FAILED'
+                status_prefix = status[:-12]
```

Comment:
Not sure where the `-12` comes from. Also, would probably be nicer (and more maintainable) to use a regex match/replace here.